clang/test/CodeGenSYCL/int_footer_with_explicit_specialization.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -fsycl-int-footer=%t.footer.h -emit-llvm %s -o -
+// RUN: FileCheck -input-file=%t.footer.h %s --check-prefix=CHECK-FOOTER
+
```

Comment:
Please add a comment here about what this test does.

Reply:
Added, thanks.